src/valkey-benchmark.c

```diff
@@ -153,6 +162,7 @@ typedef struct _client {
                            such as auth and select are prefixed to the pipeline of
                            benchmark commands and discarded after the first send. */
     int prefixlen;      /* Size in bytes of the pending prefix commands */
+    int flags;
```

Comment:
Is there any benefit to bitwise-or flags compared to individual 1-bit integers?

```suggestion
    unsigned paused : 1;
    unsigned reuse : 1;
```